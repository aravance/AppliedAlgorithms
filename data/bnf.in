x = 4
PRINT(x)
z = x + y
PRINT(z)
z : y
y = x + x
z = x + y + y - y
PRINT(z)
PRINT(y)
PRINT (X)
PRINT (x)
x = x
y =
w = 9
y = 9
y = y + x - z - z
x = x + x - x - x - x
PRINT(x)
PRINT(y)
PRINT(z)
HALT
