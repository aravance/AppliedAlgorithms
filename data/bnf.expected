x = 4
PRINT(x)                       4
z = x + y                      y is undefined
PRINT(z)                       z is undefined
z : y                          Bad syntax
y = x + x
z = x + y + y - y
PRINT(z)                       12
PRINT(y)                       8
PRINT (X)                      Bad syntax
PRINT (x)                      4
x = x
y =                            Bad syntax
w = 9                          Bad syntax
y = 9
y = y + x - z - z
x = x + x - x - x - x
PRINT(x)                       -4
PRINT(y)                       -11
PRINT(z)                       12
